### ISSUE TITLE
Make imports work with `moduleResolution: "node"`

### ISSUE THREAD

**Issue by pomber** [code-hike/codehike#437]
On a new Next.js app, when you do `import { remarkCodeHike, recmaCodeHike } from "codehike/mdx"` typescript complains:

```
Cannot find module 'codehike/mdx' or its corresponding type declarations.
  There are types at 'node_modules/codehike/dist/mdx.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.ts(2307)
```

To fix it you need to change `moduleResolution` to `bundler` in the `tsconfig.json`. Would be nice to not have to do that.

**Comment by gurkerl83** [comment]
I see this warning also. Currently I use neither of `node16`, `nodenext`, or `bundler` but simply `Node`. The build works but for sure this is a type export problem in codehike.

**Comment by pomber** [comment]
More info: https://github.com/andrewbranch/example-subpath-exports-ts-compat

**Comment by pomber** [comment]
Released in `v1.0.1`
